### PR TITLE
Correct repository URL

### PR DESCRIPTION
**Pull request by keithc-ca**
Fixes: #6291.

### CODE REVIEW

**Comment by mstoodle** [comment]
jenkins build zos

**Comment by keithc-ca** [comment]
Previous attempts failed because
> The unauthenticated git protocol on port 9418 is no longer supported.

So it seems something like this is needed, but now it fails
> Host key verification failed.

I'm not sure what is involved in that verification step. Do we need github.com to be listed in `known_hosts` (or perhaps the entry is stale)?

**Comment by mstoodle** [comment]
Progress:
```
 > git fetch --no-tags --force --progress -- git@github.com:eclipse/omr.git +refs/pull/6292/merge:refs/remotes/origin/pr/6292/merge # timeout=10
ERROR: Error fetching remote repo 'origin'
hudson.plugins.git.GitException: Failed to fetch from git@github.com:eclipse/omr.git
	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:1001)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1242)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1302)
	at org.jenkinsci.plugins.workflow.steps.scm.SCMStep.checkout(SCMStep.java:129)
	at org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition.create(CpsScmFlowDefinition.java:154)
	at org.jenkinsci.plugins.workflow.cps.CpsScmFlowDefinition.create(CpsScmFlowDefinition.java:68)
	at org.jenkinsci.plugins.workflow.job.WorkflowRun.run(WorkflowRun.java:310)
	at hudson.model.ResourceController.execute(ResourceController.java:99)
	at hudson.model.Executor.run(Executor.java:431)
Caused by: hudson.plugins.git.GitException: Command "git fetch --no-tags --force --progress -- git@github.com:eclipse/omr.git +refs/pull/6292/merge:refs/remotes/origin/pr/6292/merge" returned status code 128:
stdout: 
stderr: Host key verification failed.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Maybe the zos build machine accounts don't have keys set up? I'll chat with @AdamBrousseau about it.

**Comment by keithc-ca** [comment]
jenkins build zlinux

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/745

**Comment by keithc-ca** [comment]
jenkins build zos

**Comment by mstoodle** [comment]
build looks like it is trying to use https repo references...

**Comment by keithc-ca** [comment]
> build looks like it is trying to use https repo references...

Yes, I'm hoping Adam has some insight into that.

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by keithc-ca** [comment]
jenkins build zos

**Comment by keithc-ca** [comment]
Progress?
```
10:38:02  hudson.plugins.git.GitException: Command "/u/jenkins/jenkins-zos/git fetch --tags --progress git@github.com:eclipse/omr.git +refs/pull/6292/merge:refs/remotes/origin/pr/6292/merge" returned status code 128:
10:38:02  stdout: FOTS1373 Permission denied (publickey).
```

**Comment by AdamBrousseau** [comment]
jenkins build xlinux

**Comment by AdamBrousseau** [comment]
Works on xlinux. Definitely a zos thing. Key is fine.

File: buildenv/jenkins/jobs/pull-requests/PullRequest-zos_390-64 [hunk 1]
```diff
@@ -7,7 +7,7 @@ pipeline {
         stage('Get Sources') {
             steps {
                 timestamps {
-                    checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[name: 'origin', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'git://github.com/eclipse/omr.git']]]
+                    checkout poll: false, scm: [$class: 'GitSCM', branches: [[name: '${sha1}']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[name: 'origin', refspec: '+refs/pull/*:refs/remotes/origin/pr/*', url: 'git@github.com:eclipse/omr.git']]]
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/pull-requests/PullRequest-zos_390-64:10]: I also just realized the file you're modifying is no longer in use in our builds. The only url affecting this is configured in each job. I think at the time we added the new file, Jan requested we leave the old files intact as he was using them on his farm.

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by AdamBrousseau** [comment]
jenkins build zos

**Comment by keithc-ca** [comment]
Thanks for your help, @AdamBrousseau.

**Review by dsouzai — APPROVED** [review]

**Comment by dsouzai** [comment]
jenkins build all

**Pull request merged**